lxd/main_forkstart.go

```diff
@@ -5,31 +5,56 @@ import (
 	"os"
 	"syscall"
 
+	"github.com/spf13/cobra"
 	"gopkg.in/lxc/go-lxc.v2"
 
 	"github.com/lxc/lxd/shared"
 )
 
-/*
- * This is called by lxd when called as "lxd forkstart <container>"
- * 'forkstart' is used instead of just 'start' in the hopes that people
- * do not accidentally type 'lxd start' instead of 'lxc start'
- */
-func cmdForkStart(args *Args) error {
-	if len(args.Params) != 3 {
-		return fmt.Errorf("Bad arguments: %q", args.Params)
+type cmdForkstart struct {
+	cmd    *cobra.Command
+	global *cmdGlobal
+}
+
+func (c *cmdForkstart) Command() *cobra.Command {
+	// Main subcommand
+	cmd := &cobra.Command{}
+	cmd.Use = "forkstart <container name> <containers path> <config>"
```

Comment:
`<containers path>` or `<container path>`?

Reply:
containers path, /var/lib/lxd/containers is expected, not /var/lib/lxd/containers/NAME